docs/components/button.md

```diff
@@ -74,12 +74,14 @@ We support `small`, `medium`, and `large` sizes, default to `medium`.
 
 ```
 
-### isOpen
+### hasCaret
 
-KButton can display a dropdown caret to the right hand side. This is useful for buttons that control dropdowns and popovers. When the prop `isOpen` is `false`, the caret will display pointing down. You can rotate the caret (active state) to point up by setting `isOpen` to `true`.
+Use this prop if you would like the KButton to display a dropdown caret to the right hand side.
 
 <Komponent :data="{ isActive: false}" v-slot="{ data }">
-  <KButton appearance="primary" :isOpen="data.isActive" @click="data.isActive = !data.isActive">I'm a button</KButton>
+  <KButton appearance="primary"
```

Comment:
No need to change this since it is wrapped in `<Komponent>`, just going to call out, be weary of breaking component definitions into multiple lines in `.md` files. The outermost HTML element MUST be on a single line or it can break the md display.

Examples:

Good:
```
<KButton appearance="primary" show-caret>
   text
</KButton>
```

```
<div>
 <KButton
    appearance="primary"
   has-caret
  >
     text
  </KButton>
</div>
```

Bad:

```
<KButton
  appearance="primary"
  show-caret
>
  text
</KButton>
```